tests/docker/ducker-rp

```diff
@@ -432,10 +435,10 @@ ducker_test() {
         fi
     done
     # Give ducktape access to our vtools library
-    extra_python_path="/opt/v/build/venv/v/lib/python3.8/site-packages/:/opt/v/build/venv/v/lib64/python3.8/site-packages/:/opt/v/tools"
+    extra_python_path="/opt/v/${v_build_dir}/venv/v/lib/python3.8/site-packages/:/opt/v/${v_build_dir}/venv/v/lib64/python3.8/site-packages/:/opt/v/tools"
```

Comment:
Is this needed now that we eliminated the vtools dependency?

Reply:
i left it there to point to the tools dir so the test could import the storage file and work as normally, but good point! I didn't update the import in `prefix_recovery_test.py`

Reply:
@0xdiba just verified that the test works